R/bs-theme-preset.R

```diff
@@ -36,10 +36,11 @@ new_bs_preset <- function(name, version, type = NULL) {
   subclass <- if (!is.null(type)) paste0("bs_preset_", type)
```

Comment:
After discussing, let's explore removing the `subclass` (and the S3 methods)?

Reply:
Done in b4eff4fedf4c2cd389be38b9c4fcf3d341a86616